guide/lib/examples/breadcrumbs_helper.rb

```diff
@@ -0,0 +1,13 @@
+module Examples
+  module BreadcrumbsHelpers
+    def breadcrumbs
+      <<~BREADCRUMBS
+        = dsfr_breadcrumbs do |b|
+          = b.breadcrumb label: "Accueil", link: "#"
```

Comment:
just thinking out loud : 
to avoid confusion / typos between `Breadcrumbs` and `Breadcrumb` maybe we could go with `Breadcrumbs` and `BreadcrumbItem` ? 

Or `BreadcrumbsNav` and `Breadcrumb` ? 

again this is a nitpick that can be ignored

Reply:
What are your thoughts on `Breadcrumbs` and `Crumb` ? I thought it looked cute 😍 

```
breadcrumbs do |b|
  b.crumb
  b.crumb
```

otherwise I'll do `BreadcrumbItem`.